src/Microsoft.DotNet.Build.Tasks/PackageFiles/tool-runtime/project.json

```diff
@@ -3,38 +3,27 @@
     "dnxcore50": {
```

Comment:
I'm OK with staging it but I do want to remove all references to dnxcore50 from things like this.